components/transferForm.tsx

```diff
@@ -393,244 +490,296 @@ function TransferForm(
+                      values.transfers.map((transfer, index) => {
+                        if (transfer.type === "contract") {
+                          return (
+                            <div
+                              className="flex flex-col md:flex-row md:space-x-4"
+                              key={(transfer as any).key.toString()}
+                              id={(transfer as any).key.toString()}
+                            >
+                              <ExecuteContractForm
+                                id={(index + 1).toString().padStart(2, "0")}
+                                getFieldProps={() =>
+                                  getFieldProps(
+                                    `transfers.${index}.values.metadata`
+                                  ).value
+                                }
+                                setField={(
+                                  lambda: string,
+                                  metadata: string
+                                ) => {
+                                  setFieldValue(
+                                    `transfers.${index}.values.lambda`,
+                                    lambda
+                                  );
+                                  setFieldValue(
+                                    `transfers.${index}.values.metadata`,
+                                    metadata
+                                  );
+                                }}
+                              />
+                              <button
+                                type="button"
+                                className={
+                                  "mx-none mt-4 block self-center justify-self-end rounded bg-primary p-1.5 font-medium text-white hover:bg-red-500 hover:outline-none focus:bg-red-500 md:mx-auto md:mt-0 md:self-end"
+                                }
+                                onClick={e => {
+                                  e.preventDefault();
+                                  remove(index);
+                                }}
+                              >
+                                Remove
+                              </button>
+                            </div>
+                          );
+                        }
+                        const withTextArea = transfer.fields.find(
+                          x => x?.kind === "textarea"
+                        )
+                          ? " flex-col md:flex-col"
+                          : "";
+
+                        return (
+                          <section key={`${transfer.type}:${index}`}>
+                            <p className="text-lg text-white">
+                              <span className="mr-2 text-zinc-500">
+                                #{(index + 1).toString().padStart(2, "0")}
+                              </span>
+                              {!transfer.fields.find(v => v.kind === "textarea")
```

Comment:
as we discussed in the meeting, would it be better if we remove the title of each TX?

Reply:
IMO I think it's better to have the titles. From a UI point of view, it helps to have a seperation between the blocks, and also I think it's convenient for new users to have title. But I agree that for us it sounds a bit useless